### ISSUE TITLE
let mutual close be called after expiry has been called

### ISSUE THREAD

**Issue by DariusParvin** [boltlabs-inc/tezos-contract#9]
mutual close enforces that the channel status is set to OPEN. It probably makes sense to also allow mutual close to be called if the channel status is set to EXPIRY.

**Comment by jakinyele** [comment]
This reminds me we should confirm we're not opening ourselves up to replay attacks with multiple ways to call the mutual close entrypoint. See: https://tezos.gitlab.io/developer/michelson_anti_patterns.html#signatures-alone-do-not-prevent-replay-attacks

**Comment by DariusParvin** [comment]
thanks for the link. We have two of the suggestions from the doc to prevent against any replay attacks with the signature:
- we're going to limit mutual close to only be called by the customer using 'SENDER'
- we're going to include cid and the contract address (from SELF_ADDRESS) in what gets signed. Either one of those should prevent it being used on another contract.

**Comment by indomitableSwan** [comment]
Why did we close this issue? I don't see a decision reached yet.

**Comment by indomitableSwan** [comment]
This is a protocol design question and should be discussed wrt the protocol spec. With issues opened in tezos-contract after a decision has been made.

**Comment by DariusParvin** [comment]
Yes, I was premature with closing this. Discussing it wrt the protocol spec makes sense. For now I'll leave the contract as before (mutual close can only happen from 'OPEN') until we make a decision.
